src/template.proto

```diff
@@ -12,6 +12,8 @@ message AnalyzeTemplate {
     string createTime = 3;
     // Template modification date
     string modifiedTime = 4;
+    // Template type language code
```

Comment:
[minor] Coming new to the project the field descriptions aren't very useful at first glace. How is the language field represented? ISO standard, "en-gb" format? I this template used as a request by the client or a response message by the server?

Given it's use [here I'd suggest 'AnalyzeTemplateParameters'](https://github.com/Microsoft/presidio-genproto/blob/master/src/analyze.proto#L14) as a more informative name.

Reply:
@itye-msft Please add documentation on the language field format. We use [ISO-639](https://en.wikipedia.org/wiki/ISO_639-1)

Reply:
@itye-msft, @lawrencegripper I agree about the ambiguous names. We added a task to redesign the entire API - structure, naming, room for future enhancements. We'll do this then as we don't want to break the API at this point in time.